webapp/components/Map/MapBaseLayerPeriodSelector.js

```diff
@@ -32,14 +40,56 @@ export const MapBaseLayerPeriodSelector = () => {
 
   const i18n = useI18n()
   const user = useUser()
-  const [state, setState] = useState({ ready: false, periods: [], periodByValue: {}, selectedPeriodValue: null })
+  const [state, setState] = useState({
+    ready: false,
+    periods: [],
+    periodByValue: {},
+    selectedPeriodValueLeft: null,
+    selectedPeriodValueRight: null,
+    leftChecked: false,
+    rightChecked: false,
+    procLeft: Proc.rgb,
+    procRight: Proc.rgb,
+  })
 
   const apiKey = User.getMapApiKey({ provider })(user)
 
-  const { ready, periods, periodByValue, selectedPeriodValue } = state
+  const {
+    ready,
+    periods,
+    periodByValue,
+    selectedPeriodValueLeft,
+    selectedPeriodValueRight,
+    leftChecked,
+    rightChecked,
+    procLeft,
+    procRight,
+  } = state
+
+  const initSideBySide = () => {
+    if (User.isSystemAdmin(user)) {
+      const layerLeft = Object.values(map._layers).find((layer) => layer.options.id !== 'side')
```

Comment:
why is the id 'side'? maybe layerPosition: 'left'

Reply:
so instead of being named as 'side' maybe is better to use left-layer and right-layer
or maybe if you like to scale the system to have a grid some positional id, like:

'left', 'center', 'right'

'00' '01' '02'
'10' '11' '12'
'20' '21' '22'